Cookbook/Technical-Documents/LabelsInPRs.md

```diff
@@ -0,0 +1,118 @@
+# How to label a Pull Request
+
+
+We use labels in our Pull Requests to be able to categorize and prioritize our work. Some specific labels are also used to trigger our CI jobs.
+
+
+## 1. The labels
+
+At the moment we have 26 labels in our repository. They can be separated into 4 groups:
```

Comment:
I would remove the `26` number (it is 25 now as the linter one is gone) as it is too prone to become outdated. I would just leaving the four groups:

```suggestion
At the moment our labels can be separated into 4 groups:
```